cfsdk/src/main/java/io/harness/cfsdk/CfClient.java

```diff
@@ -95,6 +95,10 @@
 
             return;
         }
+
+        final String environmentID = authInfo.getEnvironmentIdentifier();
+        final String clusterID = authInfo.getClusterIdentifier();
```

Comment:
I am not sure if this is using cluster or clusterIdentifier behind the scene.

Reply:
This is the entry point for passing cluster further. Same for the environment.